src/main/java/net/atos/zac/app/admin/converter/RestBetrokkeneKoppelingenConverter.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.admin.converter;
+
+import net.atos.zac.admin.model.BetrokkeneKoppelingen;
+import net.atos.zac.admin.model.ZaakafhandelParameters;
+import nl.info.zac.app.admin.model.RestBetrokkeneKoppelingen;
+
+public final class RestBetrokkeneKoppelingenConverter {
+    public RestBetrokkeneKoppelingen convert(final BetrokkeneKoppelingen betrokkeneKoppelingen) {
```

Comment:
```suggestion
    public static RestBetrokkeneKoppelingen convert(final BetrokkeneKoppelingen betrokkeneKoppelingen) {
```

Everything that can be static should be static (i.e. pure functions), but again, why not Kotlin?